index.js

```diff
@@ -199,6 +211,10 @@ app.on('ready', () => {
       }
     });
 
+    win.on('focus', () => {
+      win.focusTime = process.uptime();
```

Comment:
I think a comment here (in the code) of why we store the focus time as opposed to keeping a variable around would be nice! (I assume it has to do with windows quitting etc)